content/source/docs/enterprise/install/installer.html.md

```diff
@@ -159,12 +159,15 @@ TFE runs `terraform plan` and `terraform apply` operations in a disposable Docke
 ![Terraform Enterprise docker image](./assets/tfe-docker-image.png)
 
 ### Requirements
- - The base image must be `ubuntu:bionic`.
+ - The base image must be `ubuntu:bionic` or an [offical RHEL7 image](https://catalog.redhat.com/software/containers/rhel7/57ea8cee9c624c035f96f3af) (e.g., `registry.access.redhat.com/rhel7`).
  - The image must exist on the Terraform Enterprise host. It can be added by running `docker pull` from a local registry or any other similar method.
+ - The software packages defined in the below exampes must be installed on the image.
```

Comment:
You know, so there's no confusion. Could we just list them here? Say something like "These are..." and list the packages they need?

```suggestion
 - The software packages defined in the examples below must be installed on the image.
```

Reply:
Since the packages are different between RHEL and Ubuntu (and there are quite a few), listing them out here felt kind of awkward. Not sure what the best approach is here?

Reply:
okay I can accept that - I'll resolve this